mlir/include/mlir/Dialect/Arith/IR/ArithBase.td

```diff
@@ -19,7 +19,11 @@ def Arith_Dialect : Dialect {
     The arith dialect is intended to hold basic integer and floating point
     mathematical operations. This includes unary, binary, and ternary arithmetic
     ops, bitwise and shift ops, cast ops, and compare ops. Operations in this
-    dialect also accept vectors and tensors of integers or floats.
+    dialect also accept vectors and tensors of integers or floats. The dialect
+    assumes unsigned integers are represented by bitvectors, and signed integers 
+    are represented by bitvectors with a two's complement representation. Unless 
+    otherwise stated, the operations within this dialect will propagate poison 
+    values (if any of its inputs are poison, then the output is poison). 
```

Comment:
```suggestion
    otherwise stated, the operations within this dialect propagate poison 
    values, i.e., if any of its inputs are poison, then the output is poison).
```

Reply:
I think we should also explain what happens for `vector` and `tensor` values -- poison is propagated elementwise.